pkg/controllers/follower/controller.go

```diff
@@ -94,38 +91,33 @@ var (
 
 // Handles for a leader or follower type.
 type typeHandles struct {
-	// <leader|follower>+federatedKind
-	name        string
-	typeConfig  *fedcorev1a1.FederatedTypeConfig
-	sourceGK    schema.GroupKind
-	federatedGK schema.GroupKind
-	informer    informers.GenericInformer
-	client      dynamic.NamespaceableResourceInterface
-	worker      worker.ReconcileWorker
+	// <leader|follower>+sourceKind
+	name       string
```

Comment:
Now that we have unified federated GK, informer, and client, do we still need `typeHandles`?